plugin/documents.py

```diff
@@ -152,8 +152,8 @@ def __del__(self) -> None:
         self._stored_region = sublime.Region(-1, -1)
         self._color_phantoms.update([])
         self.view.erase_status(AbstractViewListener.TOTAL_ERRORS_AND_WARNINGS_STATUS_KEY)
-        self._clear_highlight_regions()
         self._clear_session_views_async()
+        self._clear_highlight_regions()
```

Comment:
The order of lines has changed here and probably there is no reason for it?
(Not a big deal but nonetheless an unnecessary change from git history perspective.)

Reply:
fdb8e7f done :)